inst/templates/ad_adeg.R

```diff
@@ -276,10 +276,18 @@ adeg <- adeg %>%
     source_var = ANRIND,
     new_var = BNRIND
   ) %>%
-  # Calculate CHG
-  derive_var_chg() %>%
-  # Calculate PCHG
-  derive_var_pchg()
+  # Calculate CHG for post-baseline records
+  # The decision on how to populate pre-baseline and baseline values of CHG is left to producer choice
```

Comment:
```suggestion
  # The decision on how to populate pre-baseline and baseline values of CHG is left to the sponsor. 
 We have chosen to not populate pre-baseline records.
```

I think sponsor rather than producer should be used.  What do you think?  If agree can you update the other comments.

Reply:
I think producer is a more inclusive term (e.g. CROs are typically vendors, not sponsors) but happy to change if all agree.

Reply:
sure producer sounds good.
```suggestion
  # The decision on how to populate pre-baseline and baseline values of CHG is left to the producer.
```